ui/tests/integration/components/secret-engine/configure-wif-test.js

```diff
@@ -0,0 +1,900 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import sinon from 'sinon';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { render, click, fillIn } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { v4 as uuidv4 } from 'uuid';
+import { hbs } from 'ember-cli-htmlbars';
+import { overrideResponse } from 'vault/tests/helpers/stubs';
+import {
+  expectedConfigKeys,
+  createConfig,
+  configUrl,
+  fillInAzureConfig,
+  fillInAwsConfig,
+} from 'vault/tests/helpers/secret-engine/secret-engine-helpers';
+import { capabilitiesStub } from 'vault/tests/helpers/stubs';
+import { WIF_ENGINES, allEngines } from 'vault/helpers/mountable-secret-engines';
+import waitForError from 'vault/tests/helpers/wait-for-error';
+
+const allEnginesArray = allEngines(); // saving as const so we don't invoke the method multiple times in the for loop
+
+module('Integration | Component | SecretEngine/ConfigureWif', function (hooks) {
```

Comment:
definitely a nit (and we don't have great standardization on this) but I find that this test name is easier to search for if it either matches the component name one to one _or_ the file structure. So like `SecretEngine::ConfigureWif` or `secret-engine/configure-wif`